src/TLuaInterpreter.cpp

```diff
@@ -3680,6 +3680,107 @@ int TLuaInterpreter::setMapWindowTitle(lua_State* L)
     return 1;
 }
 
+// Documentation: https://wiki.mudlet.org/w/Manual:Lua_Functions#setupMapSymbolFont
+int TLuaInterpreter::setupMapSymbolFont(lua_State* L)
```

Comment:
```suggestion
int TLuaInterpreter::setMapSymbolFont(lua_State* L)
```

Just so it's consistent with other calls

Reply:
Actually I intentionally used "setup" because when you are messing with fonts it isn't always as simple as a *set* - as you don't always get what you ask for (AFAICT) - instead the underlying Qt library takes a look at what you asked for and sees if it can come up with something (what it decides actually goes into a `QFontInfo` class - how well that matches to the original `QFont` can vary from system to system even with the same OS as they will not have the same fonts installed in the same order in the same locations.)

Reply:
I know, but the fine difference will be lost on people and consistency is better.